src/components/biggive-carousel/biggive-carousel.tsx

```diff
@@ -0,0 +1,108 @@
+import { Component, Prop, Element, h } from '@stencil/core';
+import { brandColour } from '../../globals/brand-colour';
+import { spacingOption } from '../../globals/spacing-option';
+
+@Component({
+  tag: 'biggive-carousel',
+  styleUrl: 'biggive-carousel.scss',
+  shadow: true,
+})
+export class BiggiveCarousel {
+  @Element() host: HTMLBiggiveCarouselElement;
+
+  @Prop() spaceBelow: spacingOption = 4;
+
+  @Prop() columnCount: number = 3;
+
+  @Prop() buttonBackgroundColour: brandColour = 'white';
+
+  @Prop() buttonIconColour: brandColour = 'primary';
+
+  currentTab = 0;
+  itemCount = 0;
+  itemWidthPx = 0;
+  min = 0;
```

Comment:
`min` is now written but never read. It looks like we don't need it.

Reply:
I have removed this